### PR TITLE
Feature/story test/pool together

### PR DESCRIPTION
**Pull request by dumebi**
Implement `pool-together` DAPP test
fixed errors

### CODE REVIEW

File: test/helpers/story/drivers/dapp-story-test-driver.ts [hunk 1]
```diff
@@ -113,7 +118,20 @@ export default class DappStoryTestDriver extends StoryTestDriver {
       }
       case 'POOL_TOGETHER': {
         const newTest = new Test('POOL_TOGETHER Lend DAPP', (async () => {
-          expect(1).to.equal(1)
+          const borrower = await getNamedSigner('borrower')
+          const loan = await LoanStoryTestDriver.getLoan(hre, borrower)
+          const { details, diamond } = loan
+          let shouldPass = true
+          const borrowedAmount = (await details.lendingToken.balanceOf(await diamond.getLoanEscrow(details.loan.id))).toString()
+          if(borrowedAmount != details.loan.borrowedAmount.toString()) shouldPass = false
```
Comment (passabilities) [inline test/helpers/story/drivers/dapp-story-test-driver.ts:126]: what specific about this check makes the `shouldPass = false`?

File: test/helpers/story/story-helpers.ts [hunk 2]
```diff
@@ -38,11 +38,11 @@ export const TREE_STRUCTURE = {
     parents: ['LOAN.TAKE_OUT'],
   },
   'DAPP.LEND.AAVE': {
-    network: STORY_NETWORKS.MAINNET,
+    network: STORY_NETWORKS.POLYGON,
     parents: ['LOAN.TAKE_OUT'],
   },
   'DAPP.LEND.POOL_TOGETHER': {
-    network: STORY_NETWORKS.MAINNET,
+    network: STORY_NETWORKS.POLYGON,
```
Comment (passabilities) [inline test/helpers/story/story-helpers.ts:45]: should be supported on both networks

**Review by passabilities — CHANGES_REQUESTED** [review]

File: test/helpers/story/drivers/dapp-story-test-driver.ts (hunk 1 above)
Comment (dumebi) [reply test/helpers/story/drivers/dapp-story-test-driver.ts:126]: what we talked about, with the borrowed amount not matching what it's depositing. as a part of that loan has already been used

File: test/helpers/story/story-helpers.ts (hunk 2 above)
Comment (dumebi) [reply test/helpers/story/story-helpers.ts:45]: oh alright

**Review by sypeer — APPROVED** [review]

**Review by mkhoury93 — APPROVED** [review]

**Pull request merged**